packages/big-design/src/components/Table/Search/styled.tsx

```diff
@@ -0,0 +1,23 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+import { FlexProps } from '../../Flex';
+import { withFlexedContainer } from '../../Flex/withFlex';
+
+export const StyledFlex = styled.div<FlexProps & { stickyHeader?: boolean }>`
```

Comment:
I just played around with these components and you don't even need this entire file. I'll suggest changes in the component to show you.